projects/backend/controllers/render.ts

```diff
@@ -0,0 +1,72 @@
+import { Request, Response } from 'express'
+import fetch from 'node-fetch'
+import { ImageSize, imageSizes, RenderedContent } from '../../Apps/common/src'
+
+const fetchRenderedArticle = async (url: string) => {
+    const response = await fetch(url, {
+        headers: {
+            Accept:
+                'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+        },
+    })
+    const responseBody = await response.text()
+    return {
+        success: response.statusText === 'OK',
+        status: response.status,
+        body: responseBody,
+    }
+}
+
+// TODO: this needs a test once we're happy with the correct format for the paths
+const replaceImageUrls = (html: string, imageSize: ImageSize): string => {
+    return html.replace(
+        /https:\/\/i.guim.co.uk\/img\//g,
+        `../../media/${imageSize}/`,
```

Comment:
I can't see a clean way to use react-native-fs to move/rename folder so we can use a clean structure.

What I wanted is:

From: `daily-edition/2020-11-05/media/tablet/media/article-img1.jpg`
To: `daily-edition/2020-11-05/media/images/media/article-img1.jpg`

So, I guess we will go with a `html` folder duplication approach for different image sizes (phone, tablet etc)

Reply:
An alternative approach would be to provide a different image bundle that will have a simpler folder structure and it will be used by the new edition app only which supports new article rendering.

The new folder structure can look like this:
`daily-edition/2020-11-05/images/media/article-img1.jpg`
`daily-edition/2020-11-05/thumbnail/media/article-img1.jpg`

And the html bundle can have clean structure like this:
`html/article1.html`
`html/article2.html`

Reply:
I think we should try doing the alternative approach you suggest - makes it so much cleaner. I'll change this PR to do that for now and then we can always do the more complex option later

Reply:
the current structure looks like this:

<img width="265" alt="Screenshot 2020-11-09 at 10 52 30" src="https://user-images.githubusercontent.com/3606555/98532505-b354d300-2279-11eb-9d92-6ae239fc690a.png">

Reply:
Yes, in the alternative approach I suggested, for the above example, we need to replace `tablet` with a static name (i.e. `images`)

Reply:
So we'd need to change it on the server, or reorganise on the client. I guess what we really want to do is just throw away the tablet/ subdirectory

Reply:
Yep. I'm trying to make that change in the archiver and will raise a PR once I can test it